kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/index.vue

```diff
@@ -570,6 +574,26 @@
     transition: box-shadow 0.25s ease;
   }
 
+  .mobile-bookmark-container {
+    display: flex;
+    max-width: 100%;
+    min-height: 141px;
+    margin: auto;
+    margin-bottom: 24px;
+    border-radius: 2px;
+    box-shadow: 0 1px 5px 0 #a1a1a1, 0 2px 2px 0 #e6e6e6, 0 3px 1px -2px #ffffff;
```

Comment:
built-in drop shadows
```scss
        @extend %dropshadow-2dp;
```

newly documented!

https://deploy-preview-287--kolibri-design-system.netlify.app/styling/#elevation